.github/workflows/pypi-cd.yml

```diff
@@ -250,7 +277,11 @@ jobs:
                   python -m venv venv
                   source venv/bin/activate
                   pip install -U pip
-                  pip install ${PIP_PRE} valkey-glide
+                  if [[ "${{ env.PIP_PRE }}" == "true" ]]; then
```

Comment:
THis should work as it was as I understand. Do you change it because it didn't work or for another reason?

Reply:
it didnt work